scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    sed -i 's/^\(.*\)$/\1 net.ifnames=0/' /bootfs/cmdline
```

Comment:
Also [this line](https://github.com/goranche/raspbian-ua-netinst/blob/unpredictable-interfaces/scripts/etc/init.d/rcS#L1073) isn't going to help either.